test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -23,6 +23,27 @@
 
 #define AM_MSG_ID 0
 
+ucs_status_t UcxContext::map_buffer(size_t length, void *address,
+                                    ucp_mem_h *memh_p, int non_blk_flag)
+{
+    ucp_mem_map_params_t mem_map_params;
+
+    mem_map_params.field_mask = UCP_MEM_MAP_PARAM_FIELD_ADDRESS |
+                                UCP_MEM_MAP_PARAM_FIELD_LENGTH; 
+    mem_map_params.address    = address;
+    mem_map_params.length     = length;
+    if (non_blk_flag) {
+        mem_map_params.field_mask |= UCP_MEM_MAP_PARAM_FIELD_FLAGS;
+        mem_map_params.flags       = non_blk_flag;
+    }
+
+    return ucp_mem_map(_context, &mem_map_params, memh_p);
```

Comment:
maybe return true/false from UcxContext::map_buffer (translate ucs_status_t to bool)
same for UcxContext::unmap_buffer

Reply:
done